lib/web_ui/lib/src/engine/compositor/util.dart

```diff
@@ -156,6 +156,12 @@ js.JsObject makeSkPaint(ui.Paint paint) {
     skPaint.callMethod('setMaskFilter', <js.JsObject>[skMaskFilter]);
   }
 
+  if (paint.imageFilter != null) {
+    final SkImageFilter skImageFilter = paint.imageFilter;
+    skPaint.callMethod(
+        'setImageFilter', <js.JsObject>[skImageFilter.skImageFilter]);
+  }
+
```

Comment:
question: does the order of method calls make a difference? If so may be we should document it.

Reply:
Order doesn't matter